x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/common.ts

```diff
@@ -8,26 +8,31 @@ import { Dictionary } from '../../../../../../common/types/common';
 
 import { JobId, DataFrameTransformWithId } from '../../../../common';
 
-export enum DATA_FRAME_RUNNING_STATE {
+export enum DATA_FRAME_TASK_STATE {
+  FAILED = 'failed',
   STARTED = 'started',
   STOPPED = 'stopped',
 }
-type RunningState = DATA_FRAME_RUNNING_STATE.STARTED | DATA_FRAME_RUNNING_STATE.STOPPED;
+
+type DataFrameTaskState =
```

Comment:
`DataFrameTaskState` is not really needed as `DATA_FRAME_TASK_STATE` itself can be used as the type for `indexer_state` and `task_state`

Reply:
Fixed in 70161e8.